src/components/Input/Input.tsx

```diff
@@ -150,47 +134,36 @@ class Input extends AutoControlledComponent<Extendable<IInputProps>, IInputState
     styles,
     variables,
   }: IRenderResultConfig<IInputProps>) {
-    const { className, input, renderIcon, renderInput, renderWrapper, type, wrapper } = this.props
+    const { input, renderIcon, renderInput, type } = this.props
     const { value = '' } = this.state
     const [htmlInputProps, rest] = partitionHTMLProps(restProps)
 
-    return Slot.create(wrapper, {
-      defaultProps: {
-        as: ElementType,
-        className: cx(Input.className, className),
-        children: (
-          <>
-            <Ref
-              innerRef={inputDomElement =>
-                (this.inputDomElement = inputDomElement as HTMLInputElement)
-              }
-            >
-              {Slot.createHTMLInput(input || type, {
-                defaultProps: {
-                  ...htmlInputProps,
-                  type,
-                  value,
-                  className: classes.input,
-                  onChange: this.handleChange,
-                },
-                render: renderInput,
-              })}
-            </Ref>
-            {Icon.create(this.computeIcon(), {
-              defaultProps: {
-                styles: styles.icon,
-                variables: variables.icon,
-              },
-              overrideProps: this.handleIconOverrides,
-              render: renderIcon,
-            })}
-          </>
-        ),
-        styles: styles.root,
-        ...rest,
-      },
-      render: renderWrapper,
-    })
+    return (
```

Comment:
let's not make this huge amount of changes just for the fact that we cannot use <Slot as={Input} />
the issue was that I forgot to add `children` as handled prop to the `Input` so they are passed as undefined to the `Slot.create` method; better fix is to just add:

```tsx
/**
 *  Used to set content when using childrenApi - internal only
 *  @docSiteIgnore
 */
children: PropTypes.node,
```
to the `propTypes` object in this file and
```tsx
children?: ReactChildren
```
to the `IInputProps` interface

@mnajdova @kuzhelov

Reply:
Yep, this solved the issue. Thanks!